restapi/src/main/java/sjchat/restapi/controllers/ChatController.java

```diff
@@ -121,9 +122,13 @@ private static User buildUserFromResponse(sjchat.users.User responseUser) {
           produces = "application/json",
           consumes = "application/json")
   @ResponseBody
-  public ResponseEntity<Chat> createChat(@RequestBody ChatRequest chatRequest) {
+  public ResponseEntity<Object> createChat(@RequestBody ChatRequest chatRequest) {
```

Comment:
You could also throw an exception and use springs exception handling feature.

http://www.baeldung.com/exception-handling-for-rest-with-spring